tests/test_diff_tally/test_diff_tally.py

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env python
+
+import glob
+import os
+try:
+    from StringIO import StringIO
+except:
+    from io import StringIO
```

Comment:
If you don't pass an object, it does return a string actually (see description of first parameter [here](http://pandas.pydata.org/pandas-docs/stable/generated/pandas.DataFrame.to_csv.html)).